ui/compontents.go

```diff
@@ -30,15 +30,23 @@ const (
 
 // layoutBalance aligns the main and sub DCR balances horizontally, putting the sub
 // balance at the baseline of the row.
-func (page pageCommon) layoutBalance(gtx layout.Context, amount string) layout.Dimensions {
+func (page pageCommon) layoutBalance(gtx layout.Context, amount string, isSwitchColor bool) layout.Dimensions {
 	// todo: make "DCR" symbols small when there are no decimals in the balance
 	mainText, subText := breakBalance(page.printer, amount)
 	return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Baseline}.Layout(gtx,
 		layout.Rigid(func(gtx C) D {
-			return page.theme.Label(values.TextSize20, mainText).Layout(gtx)
+			label := page.theme.Label(values.TextSize20, mainText)
+			if isSwitchColor {
+				label.Color = page.theme.Color.DeepBlue
+			}
+			return label.Layout(gtx)
```

Comment:
Why is the colour for the text being done specifically here, shouldn't it be affected like other text colours in the app?

Reply:
Because the text in here alway display black color independent dark or light mode
![image](https://user-images.githubusercontent.com/33905113/119283394-11219900-bc67-11eb-93ad-24714a33e181.png)